wm.go

```diff
@@ -14,13 +14,13 @@ type WindowManager interface {
 }
 
 // Stack describes an ordered list of windows.
-// The order of the windows in this list matches the stacking order on screen.
-// TopWindow() returns the 0th element with each item after that being stacked below the previous.
```

Comment:
Wow, I forgot that we defined it this way around.
Are we certain that reversing it for all platforms is the right thing to do?

Reply:
I can't see any reason why it wouldn't be fine for any other platform -- I'm not certain though.  However I'm less certain that having it top to bottom is the right thing to do.  So far we have one implementation - X and it says bottom to top.  That should be fine for Wayland from what I can tell so far too...  I would say between the two there isn't a lot of certainty of which one is better - but the one concrete thing we have and know is how the one platform we have wants it bottom to top and from a code standup that means less conversions and slice walking so I think that tips that scales for me.